.github/workflows/container-version.yaml

```
@@ -0,0 +1,25 @@
+on:
+  schedule:
+    - cron: '37 13 * * *'
+name: busybox-update workflow
+jobs:
+  checkVersionAndCreatePR:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run busybox updater
+        run: |
+          chmod +x ./scripts/busybox-updater.sh
+          ./scripts/busybox-updater.sh
+        shell: bash
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          signoff: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: 'Updates busybox SHA'
+          title: '[busybox] Updates busybox SHAs'
```

Nit: PR titles usually follow convention `<component>: <title>`, maybe
```suggestion
          title: '*: Updates Prometheus BusyBox image SHAs'
```